dataplane/Cargo.toml

```diff
@@ -15,7 +15,7 @@ iptrie = { workspace = true }
 default-net = { workspace = true }
 dyn-iter = { workspace = true }
 mio = { workspace = true, features = ["os-ext", "net"] }
-net = { workspace = true, features = ["serde"] }
+net = { workspace = true, features = ["serde", "test_buffer"] }
```

Comment:
What do you need this for, given that this comes as the last commit in the PR? :thinking: